themes/default/content/docs/intro/languages/java.md

```diff
@@ -10,7 +10,9 @@ menu:
 
 <img src="/logos/tech/java.svg" align="right" width="90" style="padding:8px; margin-top: -64px">
 
-Pulumi supports writing your infrastructure as code using the Java language. Java 11 or later is required. We support running Maven Pulumi programs, Gradle Pulumi programs, and Pulumi programs packaged as JAR files.
+Pulumi supports writing your infrastructure as code using the Java language running on any [Supported version](https://www.oracle.com/java/technologies/java-se-support-roadmap.html) after Java 11.
```

Comment:
```suggestion
Pulumi supports writing your infrastructure as code using the Java language running on any [supported version](https://www.oracle.com/java/technologies/java-se-support-roadmap.html) after Java 11.
```

Reply:
"After 11" reads a bit ambiguously to me. (My first thought was, _Oh, I thought we supported 11 also. Do we only support 12 and up?_) If we do mean "11 or later", we should probably say that.

Reply:
It's 11 or later. So "Pulumi supports writing your infrastructure as code using the Java language running on any supported version after Java 11 or later." ?